driver-sync/src/test/functional/com/mongodb/client/ClientSideEncryptionCorpusTest.java

```diff
@@ -197,8 +195,10 @@ public void testCorpus() throws IOException, URISyntaxException {
                 BsonValue encryptedValue = clientEncryption.encrypt(value, opts);
                 fieldDocument.put("value", encryptedValue);
                 corpusCopied.append(field, fieldDocument);
-            } catch (Exception e) {
-                assertFalse(allowed);
+            } catch (MongoException e) {
+                if (allowed) {
```

Comment:
Easier to debug if we just throw here and let the exception bubble up

Reply:
Oh nice.